src/main/java/net/fabricmc/loader/impl/util/Arguments.java

```diff
@@ -25,7 +25,10 @@
 import java.util.Map;
 
 public final class Arguments {
+	// set the game version for the builtin game mod/dependencies, bypassing auto-detection
 	public static final String GAME_VERSION = "fabric.gameVersion";
+	// additional mods to load (path separator separated paths, @ prefix for meta-file with each line referencing an actual file)
+	public static final String ADD_MODS = "fabric.addMods";
```

Comment:
These fields should be like
```suggestion
	public static final String GAME_VERSION = SystemProperties.GAME_VERSION;
	// additional mods to load (path separator separated paths, @ prefix for meta-file with each line referencing an actual file)
	public static final String ADD_MODS = SystemProperties.ADD_MODS;
```